nilearn/datasets/func.py

```diff
@@ -1275,3 +1277,192 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
         X, y, mask_img = _load_mixed_gambles(map(nibabel.load, data.zmaps))
         data.zmaps, data.gain, data.mask_img = X, y, mask_img
     return data
+
+
+def fetch_megatrawls_netmats(data_dir=None, dimensionality=[25, 50, 100, 200, 300],
```

Comment:
You should probably use tuples for default arguments. Using mutable arguments as defaults is one of the Python gotchas. See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments for example.